web/src/settings_org.js

```diff
@@ -639,6 +639,38 @@ export function discard_realm_default_property_element_changes(elem) {
     update_dependent_subsettings(property_name);
 }
 
+function discard_realm_settings_subsection_changes($subsection) {
```

Comment:
Have kept these functions different for realm, stream, group, etc. like we have different functions for discarding single setting.